site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2437,6 +2439,53 @@ public function getGradedComponentsCountByGradingSections($g_id, $sections, $sec
         return $return;
     }
 
+
+    /**
+     * Returns an array of Verified components for each Section
+     *
+     * @param  array<int>  $sections
+     * @return array<int|string,int>
+     */
+    public function getVerifiedComponentsCountByGradingSections(string $g_id, array $sections, string $section_key, bool $is_team): array {
+        $unit = "users";
+        $id = "user_id";
+        if ($is_team) {
+            $unit = "gradeable_teams";
+            $id = "team_id";
+        }
+
+        $this->course_db->query(
+            "
+            SELECT 
+                " . $unit . "." . $section_key . ", COUNT(" . $unit . "." . $id . ") as verified_components_count
```

Comment:
Since this is a double-quoted string, it would be more readable to put the variables in the string directly, instead of cluttering the query with string concatenations everywhere.

Reply:
I kinda didn't realize format strings were a thing in PHP, my bad